### DISCUSSION TITLE
Will dependabot still scan archived repo?

### DISCUSSION THREAD

**Discussion by Dingjie-Daniel-Yang** [community/community#24663] [Apps, API and Webhooks]
<p>Hi Team,</p>
<p>We turned on Dependabot at organization level to detect the vulnerabilities in the dependency. We want to archive some repos but <strong>we would like to confirm whether the Dependabot will still scan the archived repos.</strong></p>
<p>Thanks<br>
Daniel</p>

**Comment by Katsute** [comment, marked as answer]
<p>Dependabot does not run on archived repos.</p>

**Comment by Dingjie-Daniel-Yang** [comment]
<p>Thank you Katsute for confirming that.</p>
